utilities/SDS-View/sds-view.py

```diff
@@ -136,6 +136,11 @@ def plotData(all_data, data_desc, freq, title):
 
         # Generate timestamps using number of data points and sampling frequency
         t = np.arange(0, len(data) / freq, 1 / freq)
+        if len(t) > len(data):
+            # We ended up with some odd round-off on the number of data points
+            # truncate timestamps back down to match the number of data points 
+            # so Matplotlib works.
```

Comment:
```suggestion
            # Truncate timestamps to match the number of data points
```
The comment in code should only describe the functionality